#!/usr/bin/env python
PACKAGE = "pure_pursuit"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("speed_multiplier", double_t, 0, "speed_multiplier", 4, 0, 8)
gen.add("max_l", double_t, 0, "max_l", 2.14, 0, 10)
gen.add("rotate_p", double_t, 0, "rotate_p", 4.25, 0, 30)
gen.add("rotate_i", double_t, 0, "rotate_i", 0.3, 0, 10)

exit(gen.generate(PACKAGE, "pure_pursuit", "PurePursuit"))